beacon-chain/das/availability.go

```diff
@@ -94,6 +94,11 @@ func (s *LazilyPersistentStore) IsDataAvailable(ctx context.Context, current pri
 	entry := s.cache.ensure(key)
 	defer s.cache.delete(key)
 	root := b.Root()
+	sumz, err := s.store.WaitForSummarizer(ctx)
+	if err == nil {
+		entry.setDiskSummary(sumz.Summary(root))
+	}
+
```

Comment:
Please do something about err != nil

Reply:
The summarizer is an optional optimization. If for whatever reason the summarizer didn't successfully init, we just don't use it. But a debug log would make sense.